crates/ln-dlc-node/src/config.rs

```diff
@@ -1,9 +1,48 @@
+use std::time::Duration;
+
```

Comment:
I think `rustfmt` should be able to fix this after our recent config changes!

Reply:
the config's clean, I must've only called dprint later. I will adjust my IDE to use the same config to prevent this in the future.